machine_learning_hep/analysis/analyzerdhadrons_mult.py

```diff
@@ -177,6 +177,25 @@ def __init__(self, datap, case, typean, period):
                                                                         None)
         self.path_for_crossmb = datap["analysis"][self.typean].get("path_for_crossmb", None)
 
+        # Take efficiencies from another analysis.
+        self.path_file_eff = datap["analysis"][self.typean].get("path_eff", None)
+        self.mult_bin_eff = datap["analysis"][self.typean].get("mult_bin_eff", None)
```

Comment:
Maybe better to include also an example of how you would put it in the DB in this PR?

Reply:
Have it in the README: https://github.com/ginnocen/MachineLearningHEP/pull/706/files/77140525195969f2a234c91fef5857ff007cb349#diff-2a1fec5d7857ca6df52463ea6a67f258R24